azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -188,6 +191,7 @@ internal class CallingViewModel(
             floatingHeaderViewModel.dismiss()
             lobbyHeaderViewModel.dismiss()
             lobbyErrorHeaderViewModel.dismiss()
+            toastNotificationViewModel.dismiss()
```

Comment:
do we also need to dismiss upperMessageBarNotificationLayoutViewModel?

Please test resume and hold scenario? On hold we hide banners and on resume we show